pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -118,6 +118,15 @@ const (
 
 	// NodeGroupNameLabel defines the label of the node group name
 	NodeGroupNameLabel = "alpha.eksctl.io/nodegroup-name"
+
+	// NATHighlyAvailable defines the highly available NAT configuration option
+	NATHighlyAvailable = "highly-available"
```

Comment:
If we rename the struct to `ClusterNAT`, this should probably become `ClusterNATHighlyAvailable`, but I'm thinking that `ClusterHighlyAvailableNAT` would be more readable?

Reply:
ClusterHighlyAvailableNAT sounds more like plain english

Reply:
exactly 👍